troposphere/static/js/components/requests/ResourceHistoryMaster.jsx

```diff
@@ -122,3 +107,5 @@ export default React.createClass({
         );
     }
 });
+
+export default subscribe(MyResourceRequestsPage, ["ProfileStore", "StatusStore", "ResourceRequestStore"]);
```

Comment:
Just being a nerd here, totally not necessary, but if the component was passed at the end, and `subscribe` returned a function with the component as an argument, it could be partially applied and reused with different components that want the same subscriptions.